### DISCUSSION TITLE
[2.8] 3xTF32: FP32 accuracy with 2x Performance

### DISCUSSION THREAD

**Discussion by hwu36** [NVIDIA/cutlass#361] [General]
In today's GTC [talk](https://reg.rainfocus.com/flow/nvidia/nvidiagtc/ap2/page/sessioncatalog/session/1627335453989001y4uG), we announced 3xTF32 as a new feature to be released with upcoming CUTLASS 2.8. Using Ampere tensor cores to emulate FP32 operations, 3xTF32 matches the accuracy of FP32 instruction with at least 2x throughput.

**Comment by thakkarV** [comment]
Congrats to everyone involved! This work is super cool

**Comment by manishucsd** [comment]
@hwu36 , Did we have an explanation of why `3xTF32` is more precise than `F32`? 

I remember and understand why `3xTF32` is more accurate than `TF32`, but cannot seem to remember if we had an explanation of `3xTF32` being more accurate than `F32` in slide 39 [here](https://www.nvidia.com/en-us/on-demand/session/gtcspring22-s41996/?playlistId=playList-d59c3dc3-9e5a-404d-8725-4b567f4dfe77)
